packages/vuetify/src/components/VList/VListGroup.ts

```diff
@@ -160,16 +161,16 @@ export default baseMixins.extend<options>().extend({
         this.genAppendIcon(),
       ])
     },
-    genItems (): VNode {
-      return this.$createElement('div', {
-        staticClass: 'v-list-group__items',
-        directives: [{
-          name: 'show',
-          value: this.isActive,
-        }],
-      }, this.showLazyContent([
-        this.$createElement('div', this.$slots.default),
-      ]))
+    genItems (): VNode[] {
```

Comment:
```suggestion
    genItems (): VNode[] | undefined {
```

If you make this `VNode[] | undefined`, you can remove the need to invoke `this.$createElement()` here: https://github.com/vuetifyjs/vuetify/pull/8823/files#diff-176112ec6d020b2c188161c378061d85R51

Reply:
I was copying Vue, v-if renders a comment when false so it can be reused.

Reply:
👍